RecoBTag/Combined/src/HeavyIonCSVTagger.cc

```diff
@@ -91,17 +91,17 @@ float HeavyIonCSVTagger::discriminator(const TagInfoHelper &tagInfo) const {
     //single value tagging var
     else {
       inputs[mva_var.name] = vars.get(mva_var.id, mva_var.default_value);
+      if (!vtxChecked) {
```

Comment:
Can the following save a few cycles?

```suggestion
      if (noTrack) {
        if (mva_var.name == "TagVarCSV_vertexMass") {
          if (inputs[mva_var.name])<0.f)
            return -1.;         
          noTrack = false;
        }
      }
      
```